hr_contract_reference/models/hr_contract.py

```diff
@@ -0,0 +1,22 @@
+# copyright 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    name = fields.Char(
+        'Contract Reference',
+        required=False,
+        readonly=True,
```

Comment:
Is it really needed to co-exist req=False and readonly=True?

Reply:
IMO name should be required, and even more if you are using this module, where you shouldnt have a contract without reference.